src/main/kotlin/ornn/racingcar/service/InputService.kt

```diff
@@ -0,0 +1,63 @@
+package ornn.racingcar.service
+
+import ornn.racingcar.resource.Strings
+import ornn.racingcar.exception.IllegalInputException
+
+class InputService {
+
+    fun readCarNames(): List<String> {
+        var carNames: List<String>
+        while (true) {
+            println(Strings.ASK_CAR_NAMES)
+            val input = readLine().toString()
+            try {
+                checkInputIsNull(input)
+                carNames = input.split(",")
+                checkCarNameMoreThanFive(carNames)
+                break
+            } catch (e: IllegalInputException) {
+                println(Strings.ERROR_MESSAGE + " " + Strings.ERROR_INFO_INVALID_INPUT)
+            }
+        }
+        return carNames
+    }
+
+    fun readPlayingTimes(): Int {
+        var input : String
+        while (true) {
+            println(Strings.ASK_HOW_MANY_TIMES_TO_PLAY)
+            input = readLine().toString()
+            try {
+                checkInputIsNull(input)
+                checkPlayTimeIsNum(input)
+                break
+            } catch (e: IllegalInputException) {
+                println(Strings.ERROR_MESSAGE + " " + Strings.ERROR_INFO_INVALID_INPUT)
+            }
+        }
+        return input.toInt()
+    }
+
+    fun checkPlayTimeIsNum (input : String) {
+        for (c in input) {
+            if (!c.isDigit()) {
+                throw IllegalInputException(Strings.ERROR_MESSAGE)
+            }
+        }
```

Comment:
```suggestion
        try {
             input.toInt()
        } catch (e: NumberFormatException){
             throw IllegalInputException(Strings.ERROR_MESSAGE)
        }
```
이렇게 바꾸면 한 메서드 안에 indent(인덴트, 들여쓰기)가 1 이면서 원하는 결과를 얻을 수 있을 것 같아요 ㅎㅎ